spring-ai-core/src/main/java/org/springframework/ai/chat/client/observation/ChatClientObservationDocumentation.java

```diff
@@ -88,7 +90,7 @@ public String asString() {
 		CHAT_CLIENT_TOOL_FUNCTION_CALLBACKS {
 			@Override
 			public String asString() {
-				return "spring.ai.chat.client.tool.functioncallbacks";
+				return "spring.ai.chat.client.tool.function.callbacks";
```

Comment:
@tzolov I made this change to align with the other attributes called "*.function.names". Is this ok?

Reply:
looks good